src/Directory.Build.targets

```diff
@@ -6,4 +6,15 @@
     <Exclude>[GraphQL.StarWars]*,[GraphQL.Harness]*,[GraphQL.Dummy]*</Exclude>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsPackable)' == 'true'">
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <WarningsAsErrors>$(WarningsAsErrors);CS1591;CS1574;CS8765;CS8601;CS8620;CS8600;CS8625;CS8767;CS8603;CS8618</WarningsAsErrors>
+    <Nullable>enable</Nullable>
```

Comment:
Do you want to exclude tests/examples for now?

Reply:
Yes, I only have so much time, and I ran out of time to do more. Ideally I would like it to apply to tests also.